lib/controllers/usuario_controller.test.js

```diff
@@ -101,6 +105,7 @@ describe('Usuario controller', () => {
       });
 
       expect(response.statusCode).toBe(201);
+      expect(MailSender.sendMail.mock.calls.length).toBeLessThanOrEqual(1);
```

Comment:
Acá podrías testear que le mandó los parámetros correctos. Al menos el mail y el subject, el body ya sería demasiado difícil de entender qué pasó si falla.

Reply:
Además estaría bueno que sea otro `it`, cosa de que si falla se note rápido que lo que no anda es el envío de mail.